invenio_communities/views/api.py

```diff
@@ -101,8 +104,39 @@ def get(self, query, sort, page, size):
             :resheader Content-Type: application/json
             :statuscode 200: no error
         """
+        if page * size >= self.max_result_window:
```

Comment:
This is only needed for Elasticsearch which has a maximum number of results that can be returned. Here we're querying the database and it's possible to return all results.